enhancements/windows-containers/health-management.md

```diff
@@ -71,8 +73,8 @@ Windows Instance Config Daemon (WICD). WICD will have the following responsibili
   based on its definition in the services ConfigMap.
 * WICD will also maintain current responsibilities of starting kubelet as a service through the `bootstrap` command. 
   WICD will fully configure kubelet in one shot, as everything required will be available in the services ConfigMap.
-* Using the configuration provided by a ConfigMap created by WMCO, WICD maintains the state of Windows Services on the 
-  instance.
+* Using the configuration provided by a ConfigMap created by WMCO, WICD maintains the state of Windows Services and 
+  the watched environment variables on the instance.
```

Comment:
You are introducing the concept of watched environment variables without defining what it is anywhere.